vault/changelog.release.2022-06-14.md

```diff
@@ -0,0 +1,79 @@
+---
+id: cpoqm8ed1q729m7lav31hhe
+title: 0.99
+desc: ''
+updated: 1655144139378
+created: 1655144139378
+---
+
+Dendron 0.99 has sprouted  🌱
+
+
+**Better Task Notes**: Get things done - task notes now have custom commands to set their status and mark them as complete.
+More information: [[Set Task Status|dendron://dendron.dendron-site/dendron.ref.commands#set-task-status]]
+
+**Obsidian Import**: Vault to vault - we now have a dedicated command to import your notes from obsidian into Dendron!
+More information: [[Import Obsidian Vault|dendron://dendron.dendron-site/dendron.ref.commands#import-obsidian-vault]]
+
+## Highlights
+- feat(structure): add set task status commands
+- feat(structure): add complete task commands
+- feat(sync): obsidian import flow
+
+## Everything Else
+- enhance(publish): make the private link colors customizable
+- enhance(views): configure creation of notes on click of graph node with `createStubs`
+- enhance(workspace): detect duplicate note id on lifecycle events
+- fix(workspace): fix duplicated panel titles
+- fix(edit): email addresses and hash symbols inside words are parsed as tags
+- fix(views): gracefully handle tree view sort error to avoid crashing the tree view
+
+## Community
+
+### General Updates
+
+#### Public Issues
+    The public data view of frequently asked questions and how to fix them is live on our [FAQ Page](https://wiki.dendron.so/notes/683740e3-70ce-4a47-a1f4-1f140e80b558/) under 'Public Issues'. We will continuously update this list with issues we seen on discord and when working with community members. Check it out and please give us feedback in the [[#feedback|dendron://dendron.dendron-site/community.discord.channels#feedback]] channel or at support@dendron.so.
+
+#### Greenhouse Series
+
+    ![[community.events.greenhouse#summary,1:#*]]
```

Comment:
looks like theres a tab/space character here? shouldn't be there

Reply:
updated